src/jdk.zipfs/share/classes/module-info.java

```diff
@@ -149,12 +149,52 @@
  *
  * <tbody>
  * <tr>
+ *   <th scope="row">accessMode</th>
+ *   <td>{@link java.lang.String}</td>
+ *   <td>null/unset</td>
+ *   <td>
+ *       A value defining the desired access mode of the file system.
+ *       ZIP file systems can be created to allow for <em>read-write</em> or
+ *       <em>read-only</em> access.
+ *       <ul>
+ *           <li>
+ *               If no value is set, the file system is created as <em>read-write</em>
+ *               if possible. Use {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} to determine the actual access mode.
+ *           </li>
+ *           <li>
+ *               If the value is {@code "readOnly"}, the file system is created
+ *               <em>read-only</em>, and {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} will always return {@code true}. Creating a
+ *               <em>read-only</em> file system requires the underlying ZIP file to
+ *               already exist.
+ *           </li>
+ *           <li>
+ *               If the value is {@code "readWrite"}, the file system is created
+ *               <em>read-write</em>, and {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} will always return {@code false}. If a writable file
+ *               system cannot be created, an {@code IOException} will be thrown
+ *               when creating the ZIP file system.
+ *           </li>
+ *           <li>
+ *               Any other values will cause an {@code IllegalArgumentException}
+ *               to be thrown when creating the ZIP file system.
+ *           </li>
+ *       </ul>
+ *       The {@code accessMode} property has no effect on reported POSIX file
+ *       permissions (in cases where POSIX support is enabled).
+ *   </td>
+ * </tr>
+ * <tr>
  *   <th scope="row">create</th>
  *   <td>{@link java.lang.String} or {@link java.lang.Boolean}</td>
  *   <td>false</td>
  *   <td>
  *       If the value is {@code true}, the ZIP file system provider creates a
- *       new ZIP or JAR file if it does not exist.
+ *       new ZIP or JAR file if it does not exist. Specifying the {@code create}
+ *       property as {@code true} with the {@code accessMode} as {@code readOnly}
```

Comment:
should we change that to `with <em>read-only</em> {@code accessMode}` or to `with the {@code accessMode} as {@code "readOnly"}`?

Reply:
I see your reasoning. In the `accessMode` property description, we have used `{@code "readOnly"}`. I will go ahead and change it to `with the {@code accessMode} as {@code "readOnly"}`.